KeychainMinder/update_authdb.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
```

Comment:
To further future-proof this we might as well point directly at /usr/bin/python2.7

Reply:
Can do 👍